universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -135,44 +157,64 @@ public ContainerTag blockOrderPanel(ProgramDefinition program, long focusedBlock
                 Styles.W_1_5,
                 Styles.BORDER_R,
                 Styles.BORDER_GRAY_200);
+    ret.with(
+        addBlocksToContainer(program, program.getNonRepeatedBlockDefinitions(), focusedBlockId, 0));
+    ret.with(
+        submitButton("Add Block")
+            .withId("add-block-button")
+            .attr(Attr.FORM, CREATE_BLOCK_FORM_ID)
+            .withClasses(Styles.M_4));
+    return ret;
+  }
 
-    for (BlockDefinition block : program.blockDefinitions()) {
+  private ContainerTag addBlocksToContainer(
```

Comment:
Suggestion (blocking): this no longer takes a container param so the method name should probably be updated. Maybe `renderBlockList`?

Reply:
done